src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenForClientParameterBuilder.cs

```diff
@@ -158,6 +158,26 @@ public AcquireTokenForClientParameterBuilder WithFmiPath(string pathSuffix)
             return this;
         }
 
+        /// <summary>
+        /// Specifies that the token refresh should only occur if the specified SHA-256 hash
```

Comment:
`Configures the SDK to not retrieve a token from the cache if it matches the SHA256 hash of the token configured. Similar to WithForceRefresh(bool) API, but instead of bypassing the cache for all tokens, the cache bypass only occurs for 1 token`